### PR TITLE
Remove Media Guide

### LINKED ISSUE #1655: Remove Logo Policy and Media Guide
### What needs to be fixed?

The canonical logo policy and media guide is now on the Rust Foundation [website](https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/).

### Page(s) Affected

https://www.rust-lang.org/policies/media-guide (and associated translations)

### Suggested Improvement

Remove the Logo Policy and Media Guide from the Rust Language website through an incoming PR

### PR DESCRIPTION
**Pull request by JoelMarcey**
The canonical logo policy and media guide is now hosted on the Rust Foundation website

https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/

Fixes #1655

### CODE REVIEW

**Comment by JoelMarcey** [comment]
Ah yes, fair enough. The same is done for the Privacy Policy. Will fix.

**Comment by Manishearth** [comment]
This still removes the translated link text. Only the policy should be removed and the link target should be updated

**Comment by pietroalbini** [comment]
Can redirect(s) be added to prevent broken links? https://github.com/rust-lang/www.rust-lang.org/blob/2d803d64cf7f562a1bb9248f3882f898c9d9c0da/src/redirect.rs#L45-L56

**Comment by JoelMarcey** [comment]
> This still removes the translated link text. Only the policy should be removed and the link target should be updated

@Manishearth What would the translated link text be used for though if the policy is not on the actual rust-lang.org site? I agree the translated link text will actually be useful if and when the Foundation gets around to translating the site - we could use that translated text as a starting point.

**Comment by Manishearth** [comment]
@JoelMarcey I'm not talking about the translated _policy_ text, I'm talking about the _link_ text in the footer: your PR switches everyone to English

**Comment by JoelMarcey** [comment]
@Manishearth Oh. Well, I just followed the precedent that was done for the Privacy Policy link change. That changes everyone to English too, right? 56/files#diff-d3d64205c4a28105d6051690404ca37daca8f1e4cedf52669e6cf60acc570649R25

 Unless I am still misunderstanding what you are referring to.

**Comment by Manishearth** [comment]
Yeah sounds like that was a mistake too. I'll fix it when I'm back on a computer

The footers of each translated website should be in that language, even if they link to English content

**Comment by Manishearth** [comment]
I fixed it for the privacy policy: https://github.com/rust-lang/www.rust-lang.org/pull/1657

File: locales/en-US/policies.ftl [hunk 1]
```diff
@@ -7,7 +7,6 @@ policies-page-description = This page links to the comprehensive set of Rust’s
 
 policies-code-of-conduct-link = Code of Conduct
 policies-licenses-link = Licenses
-policies-media-guide-link = Logo Policy and Media Guide
```
Comment (Manishearth) [inline locales/en-US/policies.ftl:10]: Basically, these entries shoudl not be removed

File: templates/policies/index.hbs [hunk 2]
```diff
@@ -12,9 +12,9 @@
     <ul>
       <li><a href="{{baseurl}}/policies/code-of-conduct">{{fluent "policies-code-of-conduct-link"}}</a></li>
       <li><a href="{{baseurl}}/policies/licenses">{{fluent "policies-licenses-link"}}</a></li>
-      <li><a href="{{baseurl}}/policies/media-guide">{{fluent "policies-media-guide-link"}}</a></li>
```
Comment (Manishearth) [inline templates/policies/index.hbs:15]: and only the URLs, not the link text, should be updated here

File: templates/policies/index.hbs [hunk 3]
```diff
@@ -12,9 +12,9 @@
...
+      <li><a href="https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/">{{fluent "policies-media-guide-link"}}</a></li>
       <li><a href="{{baseurl}}/policies/security">{{fluent "policies-security-link"}}</a></li>
-      <li><a href="{{baseurl}}/policies/privacy">{{fluent "policies-privacy-link"}}</a></li>
```
Comment (Manishearth) [inline templates/policies/index.hbs:17]: this bit still needs to be fixed

**Comment by Manishearth** [comment]
A pull request to the deploy branch needs to be done, after reviewing the staging site.

I'm pretty busy these days and most of the other people involved in the website aren't that active so it's not something that occurs regularly; but you can ask for a deploy any time (or make the PR yourself). 

I'm doing it here: https://github.com/rust-lang/www.rust-lang.org/pull/1660

**Comment by JoelMarcey** [comment]
Thanks @Manishearth for helping me understand that it is not an automatic process and for kicking off a deploy as well.

**Comment by Manishearth** [comment]
Yeah, it used to be automatic, but it was easy to break the website, so there's a manual review step now

**Pull request merged**